src/cf.data.pre

```diff
@@ -1297,8 +1298,8 @@ ENDIF
 	  # their parameter syntax and username matching algorithm.
 
 	acl aclname proxy_auth_regex [-i] username_pattern ...
-	  # perform http authentication challenge to the client and regex match
-	  # supplied username [slow]
+	  # perform http authentication challenge to the client and
+	  # POSIX extended regexp supplied username [slow]
```

Comment:
AFAICT, some words appear to be missing/dropped from the second half of the new sentence:

```suggestion
	  # POSIX extended regexp match on supplied username [slow]
```

Reply:
thanks, fixing